ebs/ebs.go

```diff
@@ -504,6 +504,50 @@ func (d *Driver) MountPoint(req Request) (string, error) {
 	return volume.MountPoint, nil
 }
 
+func (d *Driver) GetVolumesInfo(ids []string) ([]map[string]string, error) {
+	d.mutex.Lock()
```

Comment:
Maybe, I have to revisit the lock logic across our driver. I think I will do lock cleanups where ever required in a different PR.